front/app/containers/Admin/projects/project/analysis/Demographics/AuthorsByDomicile.tsx

```diff
@@ -0,0 +1,174 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+
+import useAuthorsByDomicile from 'api/analysis_stats/useAuthorsByDomicile';
+import useAnalysisFilterParams from '../hooks/useAnalysisFilterParams';
+import {
+  BarChart,
+  Bar,
+  XAxis,
+  Tooltip,
+  ResponsiveContainer,
+  Cell,
+  LabelList,
+} from 'recharts';
+
+import {
+  Box,
+  colors,
+  defaultStyles,
+  Title,
+  Text,
+} from '@citizenlab/cl2-component-library';
+import useLocalize from 'hooks/useLocalize';
+import useUserCustomFieldsOptions from 'api/user_custom_fields_options/useUserCustomFieldsOptions';
+import { xor } from 'lodash-es';
+import { updateSearchParams } from 'utils/cl-router/updateSearchParams';
+
+// Custom tooltip is rendered when the user hovers over a bar
+const CustomTooltip = ({ payload }: { payload?: any }) => {
+  const total = payload[0]?.payload?.total;
+  const filtered = payload[0]?.payload.filtered;
+
+  return (
+    <Box
+      bg={colors.white}
+      boxShadow={defaultStyles.boxShadowHoverBig}
+      p="12px"
+      borderRadius="3px"
+      w="120px"
+    >
+      <Title variant="h6" my="4px">
+        {payload[0]?.payload?.name}
+      </Title>
+      <Text m="2px">
+        {filtered === total ? (
+          <>{total}</>
+        ) : (
+          <>
+            {filtered} / {total}
+          </>
+        )}
+      </Text>
+    </Box>
+  );
+};
+
+type Props = {
+  customFieldId: string;
+};
+
+const AuthorsByDomicile = ({ customFieldId }: Props) => {
+  const localize = useLocalize();
+  const { analysisId } = useParams() as { analysisId: string };
+  const filters = useAnalysisFilterParams();
+  const { data: totalAuthorsByDomicile } = useAuthorsByDomicile({
+    analysisId,
+    queryParams: {},
+  });
+  const { data: filteredAuthorsByDomicile } = useAuthorsByDomicile({
+    analysisId,
+    queryParams: filters,
+  });
+  const { data: options } = useUserCustomFieldsOptions(customFieldId);
+
+  const filterKey = `author_custom_${customFieldId}`;
+
+  const chartData =
+    totalAuthorsByDomicile &&
+    Object.entries(totalAuthorsByDomicile?.data.attributes.series.users).map(
+      ([optionId, count]) => {
+        const option = options?.data.find((o) => o.id === optionId);
+
+        let name;
+        let shortName;
+        let filtered;
+        let optionKey;
+
+        if (optionId === '_blank') {
+          name = 'Unknown';
```

Comment:
Could you add all Demographic translations to a`translation.ts` file like we did for the copy PR? 🙏

Reply:
Yes, was thinking to do that after you've merged your branch. There is other copy in there too.